### PR TITLE
add logging in UI Atomation Infrastucture

### PR DESCRIPTION
**Pull request by ayushmeghwani**

### CODE REVIEW

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/App.java [hunk 1]
```diff
@@ -88,8 +91,10 @@ public void install() {
         //TODO: make it build time configurable to specify the installer that should be used.
         // Ideally we can specify different installers on app basis
         if (appInstaller instanceof LocalApkInstaller && !TextUtils.isEmpty(localApkFileName)) {
+            Logger.i(TAG, "Installing the "+ this.appName + "from local..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/App.java:94]:
```suggestion
            Logger.i(TAG, "Installing the " + this.appName + " from local apk..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/App.java [hunk 2]
```diff
@@ -88,8 +91,10 @@ public void install() {
...
             appInstaller.installApp(localApkFileName);
         } else {
+            Logger.i(TAG, "Installing the "+ this.appName + "from Play Store..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/App.java:97]:
```suggestion
            Logger.i(TAG, "Installing the " + this.appName + " from Play Store..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/AzureSampleApp.java [hunk 3]
```diff
@@ -100,6 +105,7 @@ public void signOutFromSingleAccountFragment() {
      * @param username the username of the account for which to confirm sign in
      */
     public void confirmSignedIn(@NonNull final String username) {
+        Logger.i(TAG, "Confirm Account with Username on the APP..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/AzureSampleApp.java:108]:
```suggestion
        Logger.i(TAG, "Confirming account with supplied username is signed in..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/TeamsApp.java [hunk 4]
```diff
@@ -101,20 +108,23 @@ private void signInWithEmail(@NonNull final String username,
...
     public void onAccountAdded() {
+        Logger.i(TAG, "Handling UI after account is added on the App..");
         UiAutomatorUtils.handleButtonClick("com.microsoft.teams:id/action_next_button");
         UiAutomatorUtils.handleButtonClick("com.microsoft.teams:id/action_next_button");
         UiAutomatorUtils.handleButtonClick("com.microsoft.teams:id/action_last_button");
     }
 
     @Override
     public void confirmAccount(@NonNull final String username) {
+        Logger.v(TAG, "confirmAccount function Not Implemented");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/TeamsApp.java:127]:
```suggestion
        Logger.w(TAG, "confirmAccount function Not Implemented");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerCompanyPortal.java [hunk 5]
```diff
@@ -222,6 +226,7 @@ public void enrollDevice(@NonNull final String username,
             );
 
             if (deviceLimitReachedDialog.exists()) {
+                Log.i(TAG, "Device limit reached for the given account..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerCompanyPortal.java:229]:
```suggestion
                Log.w(TAG, "Device limit reached for the given account..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java [hunk 6]
```diff
@@ -91,6 +96,7 @@ public void performDeviceRegistration(@NonNull final String username,
 
     @Override
     public void performSharedDeviceRegistration(String username, String password) {
+        Logger.i(TAG,"Perform Shared Device Registration for the given account..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java:99]:
```suggestion
        Logger.i(TAG, "Perform Shared Device Registration for the given account..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java [hunk 7]
```diff
@@ -117,13 +123,15 @@ public void performSharedDeviceRegistration(String username, String password) {
...
 
+        Logger.i(TAG, "Handle prompt of AAD login page for Shared Device Registration..");
         // handle AAD login page
         aadPromptHandler.handlePrompt(username, password);
 
         postJoinConfirmHelper(username);
     }
 
     private void performDeviceRegistrationHelper(@NonNull final String username) {
+        Logger.i(TAG,"Execution of Helper for Device Registration..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java:134]:
```suggestion
        Logger.i(TAG, "Execution of Helper for Device Registration..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java [hunk 8]
```diff
@@ -227,6 +238,7 @@ public void enableBrowserAccess() {
 
     @Override
     public DeviceAdmin getAdminName() {
+        Logger.i(TAG,"Get Admin name..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java:241]:
```suggestion
        Logger.i(TAG, "Get Admin name..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java [hunk 9]
```diff
@@ -166,6 +175,7 @@ private void postJoinConfirmHelper(@NonNull final String expectedUpn) {
     @Nullable
     @Override
     public String obtainDeviceId() {
+        Logger.i(TAG,"Obtain Device Id..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java:178]:
```suggestion
        Logger.i(TAG, "Obtain Device Id..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java [hunk 10]
```diff
@@ -200,6 +210,7 @@ public String obtainDeviceId() {
 
     @Override
     public void enableBrowserAccess() {
+        Logger.i(TAG,"Enable Browser Access..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java:213]:
```suggestion
        Logger.i(TAG, "Enable Browser Access..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java [hunk 11]
```diff
@@ -237,6 +249,7 @@ public void handleFirstRun() {
 
     @Nullable
     public String getAccountUpn() {
+        Logger.i(TAG,"Get Account Upn..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java:252]:
```suggestion
        Logger.i(TAG, "Get Account Upn..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/MicrosoftStsPromptHandler.java [hunk 12]
```diff
@@ -51,6 +56,7 @@ public MicrosoftStsPromptHandler(
                 loginComponentHandler,
                 parameters
         );
+        Logger.i(TAG, "Initializing IMicrosoft Sts Prompt Handler..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/MicrosoftStsPromptHandler.java:59]:
```suggestion
        Logger.i(TAG, "Initializing Microsoft Sts Prompt Handler..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java [hunk 13]
```diff
@@ -134,6 +146,7 @@ public static void enableAutomaticTimeZone() {
      * Disable automatic time zone on the device.
      */
     public static void disableAutomaticTimeZone() {
+        Logger.i(TAG,"Disable automatic time zone on the device..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java:149]: you could also add a log above in the `enableAutomaticTimeZone` method

**Review by shahzaibj — COMMENTED** [review]
Left some comments....there are some code formatting issues. I pointed out some but there are probably more so please check for those

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/App.java (hunk 1 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/App.java]: added

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/App.java (hunk 2 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/App.java]: updated.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/AzureSampleApp.java (hunk 3 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/AzureSampleApp.java]: updated.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/TeamsApp.java (hunk 4 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/TeamsApp.java]: updated.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerCompanyPortal.java (hunk 5 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerCompanyPortal.java]: updated.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java (hunk 6 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java]: updated.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java (hunk 7 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java]: updated.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java (hunk 8 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java]: updated.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java (hunk 9 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java]: updated.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java (hunk 10 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java]: updated.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java (hunk 11 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java]: updated.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java (hunk 13 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java]: added

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/MicrosoftStsPromptHandler.java (hunk 12 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/MicrosoftStsPromptHandler.java]: added.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java [hunk 14]
```diff
@@ -41,13 +42,16 @@
...
+    private final static String TAG = UiAutomatorUtils.class.getSimpleName();
+
     /**
      * Obtain an instance of the UiObject for a given resource id.
      *
      * @param resourceId the resource id of the element to obtain
      * @return the UiObject associated to the supplied resource id
      */
     public static UiObject obtainUiObjectWithResourceId(@NonNull final String resourceId) {
+        Logger.i(TAG, "Obtain an instance of the UiObject for a given resource id..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:54]: let's also log the resource id in the log message

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java [hunk 15]
```diff
@@ -67,6 +71,7 @@ public static UiObject obtainUiObjectWithResourceId(@NonNull final String resour
     @NonNull
     public static UiObject obtainUiObjectWithResourceIdAndEnabledFlag(@NonNull final String resourceId,
                                                                       final boolean enabled) {
+        Logger.i(TAG, "Obtain an instance of an enabled UiObject for the resource Id..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:74]: let's also log the resource id and enabled value in the log message

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java [hunk 16]
```diff
@@ -87,6 +92,7 @@ public static UiObject obtainUiObjectWithResourceIdAndEnabledFlag(@NonNull final
      */
     @NonNull
     public static UiObject obtainEnabledUiObjectWithExactText(@NonNull final String text) {
+        Logger.i(TAG, "Obtain an instance of an enabled UiObject for the given text..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:95]: let's also log the text in the log message

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java [hunk 17]
```diff
@@ -106,6 +112,7 @@ public static UiObject obtainEnabledUiObjectWithExactText(@NonNull final String
      * @return the UiObject associated to the supplied text
      */
     public static UiObject obtainUiObjectWithText(@NonNull final String text) {
+        Logger.i(TAG, "Obtain an instance of the UiObject for the given text..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:115]: let's also log the text in the log message

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java [hunk 18]
```diff
@@ -124,6 +131,7 @@ public static UiObject obtainUiObjectWithText(@NonNull final String text) {
      */
     public static UiObject obtainUiObjectWithResourceIdAndText(@NonNull final String resourceId,
                                                                @NonNull final String text) {
+        Logger.i(TAG, "Obtain an instance of an UiObject for the given resource Id and text..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:134]: let's also log the resource id in the log message

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java [hunk 19]
```diff
@@ -144,6 +152,7 @@ public static UiObject obtainUiObjectWithResourceIdAndText(@NonNull final String
      */
     public static UiObject obtainUiObjectWithTextAndClassType(@NonNull final String text,
                                                               @NonNull Class className) {
+        Logger.i(TAG, "Obtain an instance of the UiObject for the given text and class name..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:155]: let's also log the text and class name in the log message

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java [hunk 20]
```diff
@@ -164,6 +173,7 @@ public static UiObject obtainUiObjectWithTextAndClassType(@NonNull final String
      */
     public static UiObject obtainChildInScrollable(@NonNull final String scrollableResourceId,
                                                    @NonNull final String childText) {
+        Logger.i(TAG, "Obtain a child element inside a scrollable view by specifying resource id and text..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:176]: let's also log the resource id in the log message

Comment (shahzaibj) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java:176]: same feedback for all other instance...when we are looking for specific things via Ui Automator...let's log what we are looking for

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java [hunk 21]
```diff
@@ -165,6 +174,7 @@ public static void copyFileToFolderInSdCard(final File file, @Nullable final Str
      * @param action action is which operation to be performed.
      */
     public static void launchIntent(@NonNull final String action) {
+        Logger.i(TAG, "Launches an activity specified by the action..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java:177]: Let's log the action in the log message

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/installer/PlayStore.java [hunk 22]
```diff
@@ -48,19 +49,22 @@
...
 public class PlayStore implements IAppInstaller {
 
+    private final static String TAG = PlayStore.class.getSimpleName();
     private static final String GOOGLE_PLAY_PACKAGE_NAME = "com.android.vending";
 
     // wait at least 5 mins for app installation from Play Store
     private static final long PLAY_STORE_INSTALL_APP_TIMEOUT = TimeUnit.MINUTES.toMillis(5);
 
     private void launchMarketPageForPackage(final String appPackageName) {
+        Logger.i(TAG, "launch Market Page For " + appPackageName + " Package..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/installer/PlayStore.java:59]:
```suggestion
        Logger.i(TAG, "Launch Market Page For " + appPackageName + " Package..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java [hunk 23]
```diff
@@ -38,7 +41,10 @@
  */
 public class AdbShellUtils {
 
+    private final static String TAG = AdbShellUtils.class.getSimpleName();
+
     public static String executeShellCommand(@NonNull final String command) {
+        Logger.i(TAG, "Execute Shell Command..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java:47]: let's log the command in the log message

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java [hunk 24]
```diff
@@ -49,6 +55,7 @@ public static String executeShellCommand(@NonNull final String command) {
     }
 
     public static String executeShellCommandAsCurrentPackage(@NonNull final String command) {
+        Logger.i(TAG, "Execute Shell Command As Current Package..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java:58]: let's log the command in the log message

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java [hunk 25]
```diff
@@ -66,6 +73,7 @@ public static String executeShellCommandAsCurrentPackage(@NonNull final String c
      * @param packageName the name of the package to install
      */
     public static void installPackage(@NonNull final String packageName) {
+        Logger.i(TAG, "Installs the supplied package on the device..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java:76]: let's log the package in the log message

Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java:76]:
```suggestion
        Logger.i(TAG, "Installing the supplied package on the device..");
```

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java [hunk 26]
```diff
@@ -78,6 +86,7 @@ public static void installPackage(@NonNull final String packageName) {
      * @param flags       the flags to use during installation of the app
      */
     public static void installPackage(@NonNull final String packageName, @NonNull String... flags) {
+        Logger.i(TAG, "Installs the supplied package on the device with the supplied flags..");
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java:89]: let's log the package name and flags in the log message

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java (hunk 21 above)
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java:177]: I don't like statements here that sound more like a javadoc type of description such as `"Launches an activity specified by the action.."`

I would rather prefer logs to be more like telling what is happening at the present time such as telling a sequence of events as they happen - something like: `"Launching an activity specified by the action: " + action`

I would give the same feedback for all other places

**Comment by shahzaibj** [comment]
@tanmaymanolkar1 Since you're going to be working on logging in the future, it would be good to get some feedback on this from you

**Comment by tanmaymanolkar1** [comment]
Sure @shahzaibj will have a peek.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java (hunk 21 above)
Comment (tanmaymanolkar1) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java:177]: A question here for @ayushmeghwani and @shahzaibj do we want all of this logging at info level, also appending package names and other metadata to the log statements, would it be better to have them at verbose level, just a thought?

**Comment by tanmaymanolkar1** [comment]
Left a comment appended to your comment @shahzaibj, if we could just get the clarity if we really need such extensive use of info, and should the appended metadata be clubbed into info level as well?

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java (hunk 14 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java]: added

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java (hunk 15 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java]: added.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java (hunk 16 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java]: added.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java (hunk 17 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java]: added.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java (hunk 18 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java]: added.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java (hunk 19 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java]: added.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java (hunk 20 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java]: added.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java (hunk 21 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java]: added.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/installer/PlayStore.java (hunk 22 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/installer/PlayStore.java]: updated.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java (hunk 23 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java]: added.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java (hunk 24 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java]: added.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java (hunk 25 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java]: added.

Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java]: Updated

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java (hunk 26 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java]: added.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java (hunk 21 above)
Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java]: According to my understanding @tanmaymanolkar1 the main purpose of logging in automation infrastructure and test cases is that when we see the automation files generated at FireBase Test Labs, we can see the state of Test Case, and these automation log files will kind of help us to see that where things went wrong.
And then these kind of logging should be done at verbose level or info level ? I will say I  have to look more on that and read some resources about it and will get back to you on that.

Comment (shahzaibj) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java]: As discussed the other day, we can move these to `info` level

Comment (ayushmeghwani) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java]: yes.

**Review by shahzaibj — APPROVED** [review]

**Pull request merged**